src/applications/pre-need-integration/config/form.jsx

```diff
@@ -6,7 +6,8 @@ import environment from 'platform/utilities/environment';
 import preSubmitInfo from 'platform/forms/preSubmitInfo';
 import { VA_FORM_IDS } from 'platform/forms/constants';
 
-import fileUploadUI from 'platform/forms-system/src/js/definitions/file';
+// import fileUploadUI from 'platform/forms-system/src/js/definitions/file';
+import { fileUploadUi } from '../../pre-need/utils/upload';
```

Comment:
Check that linting warning for this line. Don't think we want to import from a different project, so I'd copy upload.js to pre-need-integration too

Reply:
Should be good to go now.